src/layouts/LandingPageLayout.res

```diff
@@ -1,21 +1,356 @@
-module Link = Next.Link
```

Comment:
From a style perspective, I haven't settled on whether I like to implement more within a layout or within a page. In this case, I would tend towards a page.